app/assets/javascripts/create.js

```diff
@@ -0,0 +1,32 @@
+$(() => {
+    createIdea()
+})
+
+var createIdea = () => {
+    $('#save-idea').click( (event) => {
+        event.preventDefault()
+        var ideaParams = {
+            title: $('#title-field').val(),
+            body:  $('#body-field').val()
+        }
+
+        $.ajax({
```

Comment:
`$.ajax` is kind of like `each` in Ruby. It's the basis of all AJAX functionality in jQuery, but there is usually an easier way to do it. Alternatively, you could write this code as follows:

```js
$.post('/api/v1/ideas'`, ideaParams)
  .then(clearFields)
  .then(clearIdeas)
  .then(loadIdeas)
  .fail(xhr => alert(xhr.responseText));
```